docs/release-notes.txt

```diff
@@ -8,6 +8,53 @@
  Version 0.13
 **************
 
+Version 0.13.4
+==============
+
+**Release Date:** September 16, 2020
+
+**Improvements**
+
+- Warn users when setting ``DistributeConfig`` on an Estimator. Configuring an Estimator via ``tf.distribute.Strategy`` can conflict with how Determined performs distributed training. With this change, Determined will attempt to catch this problem and surface an error message in the experiment logs. An Estimator can still be configured with an empty ``DistributeConfig`` without issue.
+
+- Support configuring default task images, Docker pull policy, and Docker registry credentials via the :ref:`master-configuration` and the :ref:`helm-config`. Note that individual task configurations will continue to overwrite these defaults.
```

Comment:
question: do the :ref: links get rendered when we publish release notes?

Reply:
Yes.

Reply:
Looks like this:

![Screen Shot 2020-09-16 at 15 10 10](https://user-images.githubusercontent.com/56557451/93397643-de4d1880-f82e-11ea-9ff7-a6a20828c8ca.png)